gn/chip/chip_test_standalone.gni

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+import("${chip_root}/gn/chip/tests.gni")
+
+assert(chip_build_tests)
+
+template("chip_test_standalone") {
```

Comment:
I'm not spotting what the difference is between chip_test_standalone and chip_test_suite. Why did we need to duplicate it?

Reply:
Done, chip_test_suite requires a Driver.cpp, but TestInetLayerMulticast does not need a Driver.cpp, modified the test to use chip_test_suite.